lua/nvim-tree.lua

```diff
@@ -660,9 +660,11 @@ local ACCEPTED_STRINGS = {
     signcolumn = { "yes", "no", "auto" },
   },
   renderer = {
+    highlight_git = { "none", "icon", "name", "all" },
```

Comment:
I was working on something similar because I needed to highlight only names but not git icons, by the time I was writing the code I discovered icons.show.git = false that did the trick for me, I'm glad that you implemented this feature.

Reply:
Many thanks @Akmadan23